### PR TITLE
[CN-360] Add info for multiple pod and service label support

### PR DESCRIPTION
**Pull request by mtyazici**

### CODE REVIEW

File: docs/modules/deploy/pages/configuring-kubernetes.adoc [hunk 1]
```diff
@@ -94,8 +94,8 @@ There are several properties to configure, all of which are optional.
...
 * `service-name`: service name used to scan only Pods connected to the given service; if not specified, then all Pods in the namespace are checked
-* `service-label-name`, `service-label-value`: service label and value used to tag services that should form the Hazelcast cluster together
-* `pod-label-name`, `pod-label-value`: pod label and value used to tag pods that should form the Hazelcast cluster together
+* `service-label-name`, `service-label-value`: service label and value used to tag services that should form the Hazelcast cluster together. Can support comma separated multiple values, an example is "label-1,label-2". Note that the number of elements in` service-label-name` and `service-label-value` must be equal. 
+* `pod-label-name`, `pod-label-value`: pod label and value used to tag pods that should form the Hazelcast cluster together. Can support comma separated multiple values, an example is "label-1,label-2". Note that the number of elements in` pod-label-name` and `pod-label-value` must be equal.
```
Comment (angelasimms) [inline docs/modules/deploy/pages/configuring-kubernetes.adoc:98]:
```suggestion
* `pod-label-name`, `pod-label-value`: pod label and value used to tag Pods, which together form the Hazelcast cluster. These properties can support multiple comma-separated values. For example: "label-1,label-2". You must use the same number of elements in `pod-label-name` as `pod-label-value`. 
```

**Review by angelasimms — CHANGES_REQUESTED** [review]
Just minor amends. Apologies if I have misunderstood anything.

**Review by angelasimms — APPROVED** [review]

**Comment by mtyazici** [comment]
@angelasimms, should I merge the PR? I have not added any links so, not sure why the dead link checker is failing.

**Comment by angelasimms** [comment]
> @angelasimms, should I merge the PR? I have not added any links so, not sure why the dead link checker is failing.

Hi @mtyazici. You can go ahead and merge. I don't think the link checker is working.

**Pull request merged**